docker-compose.yml

```diff
@@ -1,43 +1,35 @@
-version: "2.4"
+version: '2.4'
 services:
   db:
     image: postgres:13.2-alpine
     ports:
-      - "5432:5432"
-    env_file:
-      - postgres/env.docker
-    healthcheck:
-      test: "pg_isready -U deploy -d openledger"
```

Comment:
In my opinion the healthcheck in local environments can hide errors that we really need to see or be aware. My proposal to move the healthcheck only to production environments is based on the fact that in local environments we don't ask docker to monitor the status of the application, instead of that, I want docker to throw me the error and be able to know what is happening immediately.

Either way, my proposal is open to suggestions.